backend/populateDB/schema.sql

```diff
@@ -454,6 +456,11 @@ begin
   insert into floods.community_crossing (community_id, crossing_id) values
     (community_id, floods_crossing.id);
 
+  -- Update the community viewport
+  update floods.community
+    set viewportgeojson = (select ST_AsGeoJSON(ST_Envelope(ST_Extent(c.coordinates))) from floods.crossing c, floods.community_crossing cc where cc.crossing_id = c.id and cc.community_id = new_crossing.community_id)
```

Comment:
I don't understand whats going in with `ST_AsGeoJSON(ST_Envelope(ST_Extent(c.coordinates))`. 

Why do we have to wrap the `ST_Extent` bounding box in `ST_Envelope` (which is another format for a bounding box?) and then convert to JSON. Asking out of ignorance, not because I know of any better way to do this.

Reply:
Good call! ST_Envelope wasn't needed, I was using it in earlier calculations and didn't think to remove it once I switched to use ST_Extent.